app/Http/Kernel.php

```diff
@@ -16,7 +16,7 @@ class Kernel extends HttpKernel
     protected $middleware = [
         // \App\Http\Middleware\TrustHosts::class,
         \App\Http\Middleware\TrustProxies::class,
-        \Fruitcake\Cors\HandleCors::class,
+        \Illuminate\Http\Middleware\HandleCors::class,
```

Comment:
Replaced missing `Fruitcake\Cors\HandleCors` with Laravel’s built-in `Illuminate\Http\Middleware\HandleCors` so the app/test suite boots cleanly on Laravel 10 without adding the Fruitcake package.